crowbar_framework/app/models/api/upgrade.rb

```diff
@@ -90,12 +90,21 @@ def adminrepocheck
           if zypper_stream["message"] =~ /^System management is locked/
             return {
               status: :service_unavailable,
-              message: I18n.t(
+              error: I18n.t(
```

Comment:
I know it's not part of your change, we have another problem, that even the normal repo check performed bellow does not return "error"...